ThirdParty/zlib-1.2.8/gzguts.h

```diff
@@ -20,6 +20,11 @@
 
 #include <stdio.h>
 #include "zlib.h"
+
+#ifdef PLATFORM_EMSCRIPTEN
+    #include "unistd.h"
+#endif
+
```

Comment:
Modifying third-party source is highly undesirable. Can this be avoided (by defining some preprocessor macros (e.g. `STDC`) from CMake).